lib/builderator/control/version.rb

```diff
@@ -32,6 +32,16 @@ class Version
       class << self
         def current
           @current ||= SCM.tags.last
+
+          if @current.nil? && Util.relative_path('VERSION').exist?
+            @current = Version.from_string(Util.relative_path('VERSION').read)
+          end
+
+          if @current.nil?
+            fail 'No current version found! Create a VERSION file or set a version tag in your SCM.'
```

Comment:
I used `fail` instead of `raise` because [existing code uses `fail`](https://github.com/rapid7/builderator/blob/fix-default-version/lib/builderator/control/version/scm.rb#L76-L77) and I wanted to keep it consistent.